src/ProfilesList.json

```
@@ -124,5 +124,6 @@
   "idevkshitij.json",
   "JesugG17.json",
   "akkash20.json",
+  "JaRKyd.json",
```

```suggestion
  "JayRKyd.json",
```
@JayRKyd there is still `y` is missing. 
It should be same as the file you created.